application/views/js_init/js_keyboard.php

```diff
@@ -147,15 +147,15 @@
     		$.post("<?php echo site_url('messages/compose_process') ?>", $("#composeForm").serialize(), function(data) {
     			$("#compose_sms_container").html(data);
     			$("#compose_sms_container" ).dialog( "option", "buttons", { "Okay": function() { $(this).dialog("destroy"); } } );
-    			setTimeout(function() {$("#compose_sms_container").dialog('destroy')} , 1500);
+    			setTimeout(function() {if ($("#compose_sms_container").hasClass('ui-dialog-content')) { $("#compose_sms_container").dialog('destroy')}} , 1500);
     		});
     		}
     	},
     	'<?php echo lang('kalkun_cancel'); ?>': function() { $(this).dialog('destroy');}
         }
       });
-    });
     $("#compose_sms_container").dialog('open');
+    });
```

Comment:
Mismatched closing braces? Shouldn't this be before line 157

Reply:
still the same error as everywhere else... You will see about 10 of these if not more. This was probably a issue that was copy/pasted everywhere.

There is no access to the element at the place it was before. So I had to move it.

JQuery Error: cannot call methods on dialog prior to initialization;

Reply:
This is a separate issue. `Mismatched closing braces? Shouldn't this be before line 157`

Reply:
I don't have this error in my js console in firefox. How do you reproduce it?

Reply:
I'm almost certain there is no mismatch as you suggest. Firefox js console says that the element could not be accessed and that open can't be made.
So I had to move it one level up.

`$("#compose_sms_container").dialog('open'); `

Or else the issue is somewhere else.

Reply:
Checking this by running. I was just reviewing based on the code.